tuf-spec.md

```diff
@@ -1091,13 +1091,11 @@ repo](https://github.com/theupdateframework/specification/issues).
   still be able to update again in the future. Errors raised during the update
   process should not leave clients in an unrecoverable state.
 
-  **5.0**. **Record the time at which the update began.** Add the update
-  workflow maximum duration T to the recorded update start time to derive the
-  fixed update expiration time. The value for T is set by the authors of the
-  application using TUF. For example, T may be tens of minutes.
-  This update expiration time will be used when checking for freeze attacks,
-  and is fixed at the beginning of the update workflow to prevent metadata
-  from expiring during an in-progress update.
+  **5.0**. **Record the time at which the update began** as the fixed update
+  expiration time.  Time is fixed at the beginning of the update workflow to
+  allow an application using TUF to effectively pause time, in order to prevent
+  metadata which is valid at the beginning of an update from expiring during
+  the update workflow.
```

Comment:
Similarly, I'm unsure about the causality implied in "pause time, in orderer to prevent". Don't we want to say something like this...?

```suggestion
  allows an application using TUF to effectively pause time, without 
  metadata, which is valid at the beginning of an update, becoming expired during
  the update workflow.
```

Reply:
How about:

> pause time, in order to ensure that metadata which has a valid expiration time at the beginning of the update does not fail an expiration check later in the update workflow

?

Reply:
I just realized that I mistook "pause time" for "pause execution", so the original phrasing does make sense and so does your update. I withdraw my suggestion and am fine with both of yours.